src/odemis/driver/lakeshore.py

```diff
@@ -233,15 +231,15 @@ def _sendQuery(self, cmd):
 
             self._serial.timeout = 1
             ans = b''
-            while ans[-1:] != b'\r':
```

Comment:
BTW, as the device only accepts LF, could you update also the docstring of sendOrder/sendQuery to say "LF" instead of "CR".